Makefile

```diff
@@ -93,15 +98,22 @@ $(TOPDIR)/bin: $(TOPDIR)
 	mkdir -p $(TOPDIR)/bin
 
 # Linux AMD64 Builds
-$(TOPDIR)/bin/amd64-linux/habitat: $(TOPDIR)/bin
+$(TOPDIR)/bin/amd64-linux/habitat: $(TOPDIR)/bin internal/frontend/build
```

Comment:
do we not still need `$(TOPDIR)/bin?

Reply:
Some pre-existing rules still rely on it